build.sh

```diff
@@ -1,4 +1,4 @@
 #!/usr/bin/env bash
 
 # This will build and make sure things compile
-node scripts/convert.js && elm-format Tachyons/Classes.elm --yes && elm-make Tachyons/Classes.elm && elm-make --docs documentation.json
+node scripts/convert.js && elm-format src/Tachyons/Classes.elm --yes && elm-make src/Tachyons/Classes.elm && elm-make --docs documentation.json
```

Comment:
I ran this file and it still needed some changes. I've been working on fixing it so that the documentation works in prod but not sure when I'll get to this.

Reply:
Ah yes, the commands needed to change. They're no longer hyphenated (except for elm-format).